packages/app/app/components/Settings/index.js

```diff
@@ -19,15 +19,19 @@ import GithubSettings from './GithubSettings';
 
 import styles from './styles.scss';
 import { withTranslation } from 'react-i18next';
+import { fetchAllFmFavorites } from '../../actions/importfavs';
+import * as FavoritesActions from '../../../app/actions/favorites';
 
 const volumeSliderColors = {
   fillColor: { r: 248, g: 248, b: 242, a: 1 },
   trackColor: { r: 68, g: 71, b: 90, a: 1 },
   thumbColor: { r: 248, g: 248, b: 242, a: 1 }
 };
-
 @withTranslation('settings')
 class Settings extends React.Component {
+  state = {
```

Comment:
State in components should never be used.
It's better to use this instead:

https://github.com/acdlite/recompose/blob/master/docs/API.md#withstate

But in this case, we don't need this variable at all. If you follow the pattern I described below, we'll have access to loading/success/error state of this data, and we'll be able to choose a message according to that.